### PR TITLE
Fix: Table of Contents - Not allowed HTML tags in title and JS minification issue.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Trello - https://trello.com/c/QYt4FKUx/291-fix-table-of-contents-not-allowed-html-tags-in-title-and-js-minification-issue.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Checked minified JS should be work for Table of content block.
- Add a Bold/Italic style to heading and it should work proper on frontend.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/table-of-contents.js [hunk 1]
```diff
@@ -85,16 +85,16 @@
...
 
-				var hash = this.hash
-				var node = $( this ). closest( '.wp-block-uagb-table-of-contents' )
+				var hash = this.hash;
+				var node = $( this ). closest( '.wp-block-uagb-table-of-contents' );
 
-				scroll = node.data( 'scroll' )
-				scroll_offset = node.data( 'offset' )
-				scroll_delay = node.data( 'delay' )
+				scroll = node.data( 'scroll' );
```
Comment (RajkiranBagal) [inline assets/js/table-of-contents.js:91]: Here if you want to add `;` the var should be there for all the variables

File: readme.txt [hunk 2]
```diff
@@ -168,6 +168,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 1.23.0 =
+* Fix: Table of Contents - Not allowed HTML tags in title and Js minification issue.
```
Comment (RajkiranBagal) [inline readme.txt:172]: Change `Js` to `JS`

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/table-of-contents.js (hunk 1 above)
Comment (AbhijitNage123) [reply assets/js/table-of-contents.js:91]: `var` is already added above here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/blob/Added-semicolon-to-toc-js/assets/js/table-of-contents.js#L3

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**